plugins/annotation/__init__.py

```diff
@@ -34,22 +34,23 @@ def resolve_input(self, ctx):
 
         _inject_annotation_secrets(ctx)
 
-        ready = request_annotations(ctx, inputs)
-        if ready:
-            _execution_mode(ctx, inputs)
+        request_annotations(ctx, inputs)
 
         view = types.View(label="Request annotations")
         return types.Property(inputs, view=view)
 
-    def resolve_delegation(self, ctx):
-        return ctx.params.get("delegate", False)
+    def resolve_execution_options(self, ctx):
```

Comment:
Only use `resolve_execution_options()` if the values are dynamic. If the values are static use `OperatorConfig` instead.

Reply:
I recommend this config, which disables resolution of execution options and speeds up things a bit.

```
        return foo.OperatorConfig(
            name="example_complex_execution",
            label="Example Complex Execution",
            allow_immediate_execution=True,
            allow_delegated_execution=True,
            default_choice_to_delegated=True,
            resolve_execution_options_on_change=False,
        )
```

Reply:
Ah good call on using `OperatorConfig`. I've done that in https://github.com/voxel51/fiftyone-plugins/pull/183/commits/843cc705fbcaef81442121d54aad1fc8bfe7c14a